page-data.php

```diff
@@ -44,11 +44,11 @@
   jQuery(function(){
     var chartView = new TJIChartView({
       chart_configs: [
-        {type: 'bar',      group_by: 'year'},
+        {type: 'bar', group_by: 'year', sort_by: {column: 'key', direction: 'asc'}},
```

Comment:
Why specify this if it's the default for bar charts?

Reply:
can remove 👍